### PR TITLE
Adding a Launch section in the Setup tutorial

### PR DESCRIPTION
**Pull request by Laurent2695**
As I was kind of lost about how to run the project after the installation (without docker), I thought about adding a dedicated "Launch" section in the setup tutorial. It also gives some tips about regrouping the command lines in a bash script.

### CODE REVIEW

File: docs/source/tutorials/launch.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Almost there ! Once you proceeded to the above installation steps, you can now
+run the following command in your terminal to run the project:
+
+.. code-block:: console
+
+    $ flask run
+
+.. note::
+
+    It is recommended to copy the above command lines (i.e. virtual environment activation,
```
Comment (jeertmans) [inline docs/source/tutorials/launch.rst:18]:
```suggestion
    It is recommended to copy the above command lines (i.e., virtual environment activation,
```
Comma after "i.e." ;-)

**Review by jeertmans — CHANGES_REQUESTED** [review]
Hi @Laurent2695 ,

Indeed, the current documentation lacks much information, and it's very nice to see external people trying to get into this project!

I think I might also add a screenshot of my terminal configuration to show what a complete setup looks like (i.e., with server running, redis runnning, webpack running, and a console using the virtualenv).

Can you maybe emphasize that the `flask run` command should be run within the virtual env (if it was used)?

Thanks!

File: docs/source/tutorials/launch.rst (hunk 1 above)
Comment (Laurent2695) [reply docs/source/tutorials/launch.rst:18]: Sure, I'm going to change this right now

**Comment by Laurent2695** [comment]
> Hi @Laurent2695 ,
> 
> Indeed, the current documentation lacks much information, and it's very nice to see external people trying to get into this project!
> 
> I think I might also add a screenshot of my terminal configuration to show what a complete setup looks like (i.e., with server running, redis runnning, webpack running, and a console using the virtualenv).
> 
> Can you maybe emphasize that the `flask run` command should be run within the virtual env (if it was used)?
> 
> Thanks!

Changes made, they are in my last commit. As I'm not used to GitHub, I don't know if I should do a new pull request or you can access them from here ?

**Comment by jeertmans** [comment]
@Laurent2695 your changes are automatically updated here ;-)

I also committed to your branch to add the small terminal configuration I mentioned.

Looks go to me now!

**Review by jeertmans — APPROVED** [review]

**Comment by jeertmans** [comment]
If seems good for you @Laurent2695 , I will merge your merge onto main ;-)

**Comment by Laurent2695** [comment]
> If seems good for you @Laurent2695 , I will merge your merge onto main ;-)

It's great ! Thanks in advance ;)

**Comment by jeertmans** [comment]
Merged!

**Pull request merged**